smartsim/_core/launcher/pbs/pbsLauncher.py

```diff
@@ -188,6 +192,17 @@ def _get_managed_step_update(self, step_ids: t.List[str]) -> t.List[StepInfo]:
 
         qstat_out, _ = qstat(step_ids)
         stats = [parse_qstat_jobid(qstat_out, str(step_id)) for step_id in step_ids]
+
+        # Fallback: if all jobs result as NOTFOUND, it might be an issue
+        # with truncated names, we resort to json format which does not truncate
+        # information
```

Comment:
I think you mentioned that you would be interested in always using the json format. I would 100% agree, could you add a ticket to do that?